linear_elasticity/include/parameter_handling.h

```diff
@@ -280,6 +286,7 @@ namespace Linear_Elasticity
         mesh_name        = prm.get("Mesh name");
         read_data_name   = prm.get("Read data name");
         write_data_name  = prm.get("Write data name");
+        flap_location    = prm.get_double("Flap location");
```

Comment:
Maybe we can add "x" in the naming, so that it is clear for the user what's actually meant by this value.